compiler/pkg/openapi_generator/generator.go

```diff
@@ -373,6 +382,32 @@ func (g *Generator) UpdateYAMLs(yamlsPath string) error {
 			if err != nil {
 				return err
 			}
+
+			/*
+				Find the old CRD for the file with the same name as the new CRD file and compare it to the new CRD spec provided.
+					yamlsPath - indicates the directory path of new crd yamls
+								Ex: the path will be `_generated/crds`
+
+					path - indicates the file path of node
+						        Ex: For the node `Config` the path will be `_generated/crds/config_config.yaml`
+
+					oldYamlsPath - indicates the directory path of the existing crd yamls
+								Ex: the path will be `tmp/old-files/old-crds`
+			*/
+			oldFilePath := oldYamlsPath + strings.TrimPrefix(path, yamlsPath)
```

Comment:
@PavitraS What will happen if a nexus node has been removed ? Looks like this will not fail ?

Reply:
Handle the node deletion scenario in the current code